src/js/backend-script.js

```diff
@@ -0,0 +1,31 @@
+function confirmNewUser(e) {
+  const $urlInput = $('#confirmUserURL'),
+    url = $urlInput.val(),
+    $token = $('input[name="csrfmiddlewaretoken"]');
+
+  $.post(
+    url,
+    {
+      csrfmiddlewaretoken: $token.val()
+    }
+  );
+}
+
+function showNewUserModal($) {
+  const $hfShowModal = $('#showNewUserModal'),
+    $newUserModal = $('#newUserModal'),
+    $newUserModalBtn = $('#newUserModalBtn'),
+    showModal = $hfShowModal ? $hfShowModal.val() : false;
+
+  if (showModal === 'True') {
+    $newUserModal.modal();
+  }
+
+  $newUserModalBtn.click(confirmNewUser);
```

Comment:
It looks like the click event gets triggered when the user focuses on the button + hits the enter key in Chrome at least, but could be worth verifying cross-browser, since the click event isn't technically supposed to get fired on-enter here.  I'm not sure if the on-enter event handling is something Bootstrap adds to its button js or not.